starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ItemsObject.java

```diff
@@ -0,0 +1,49 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by colinbradley on 10/25/16.
+ */
+
+public class ItemsObject {
+    String mItemName, mDescription, mType;
+    String mPrice;
```

Comment:
I suggesting making these `private` - when you don't specify, the default is "package private" which means code in any class in the same package/folder can access & modify these variables.